util/nrt_diff/nrtDiff.py

```diff
@@ -18,7 +18,6 @@
 import imp
 import itertools
 
-from xml.dom import minidom
```

Comment:
I have a doubt: did you make sure that those dependencies are inherited with iidmDiff import? otherwise it will not work

Reply:
Yes, imports are inherited. I checked it on dynawo nrt, dynawo-rte nrt and dynawo-interface nrt

Reply:
Same goes with settings

Reply:
yes